src/pypnusershub/routes.py

```diff
@@ -223,12 +223,20 @@ def authorize(provider="local_provider"):
     return redirect(current_app.config["URL_APPLICATION"])
 
 
-def insert_or_update_organism(organism):
+def insert_or_update_organism(organism) -> dict:
     """
-    Insert a organism
+    Insert a organism into the database.
```

Comment:
```suggestion
    Insert a organism 
```
Ca semble plutôt logique que ça soit dans la base

Reply:
En effet,

Modifié.